### PR TITLE
Update DynamoDB Docs, replaced shell with aws cli command

### PR DESCRIPTION
**Pull request by hamburml**
issue https://github.com/quarkiverse/quarkus-amazon-services/issues/246

### CODE REVIEW

File: docs/modules/ROOT/pages/amazon-dynamodb.adoc [hunk 1]
```diff
@@ -34,32 +34,21 @@ The easiest way to start working with DynamoDB is to run a local instance as a c
...
+This starts a DynamoDB instance that is accessible on port `4566`.
+You can check if it's working with `aws dynamodb list-tables --endpoint-url http://localhost:4566`.
 
-Have a look at the https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html[Setting Up DynamoDB Local guide] for other options to run DynamoDB.
-
-Open `http://localhost:8000/shell` in your browser.
-
-Copy and paste the following code to the shell and run it:
-[source,js,subs="verbatim,attributes"]
 ----
```
Comment (gsmet) [inline docs/modules/ROOT/pages/amazon-dynamodb.adoc:43]:
```suggestion
[source,json]
----
```

File: docs/modules/ROOT/pages/amazon-dynamodb.adoc [hunk 2]
```diff
@@ -34,32 +34,21 @@ The easiest way to start working with DynamoDB is to run a local instance as a c
...
-};
+{
+    "TableNames": []
+}
+----
 
-dynamodb.createTable(params, function(err, data) {
-    if (err) ppJson(err);
-    else ppJson(data);
+You should see this in your terminal. That's it, DynamoDB runs locally.
```
Comment (gsmet) [inline docs/modules/ROOT/pages/amazon-dynamodb.adoc:49]:
```suggestion
You should see the output above in your terminal. That's it, DynamoDB runs locally.
```

File: docs/modules/ROOT/pages/amazon-dynamodb.adoc [hunk 3]
```diff
@@ -68,6 +57,8 @@ For more information, see https://docs.aws.amazon.com/amazondynamodb/latest/deve
 
 We recommend to use the AWS CLI to provision the table:
 
+=== Create the QuakrusFruits Table
```
Comment (gsmet) [inline docs/modules/ROOT/pages/amazon-dynamodb.adoc:60]:
```suggestion
=== Create the QuarkusFruits Table
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks, I added a few suggestions.

Would you be interested in having a look at the other guides? They all have the same issue.

**Comment by hamburml** [comment]
I'll try, but I did not used the other aws parts of this extension. Should be mostly a port replacement I think.

**Review by gsmet — APPROVED** [review]
I rebased and squashed. Let's get this in!

**Comment by gsmet** [comment]
@scrocquesel I'm merging this but I wonder if we should adjust the documentation now that we have the dev services?

**Comment by gsmet** [comment]
Thanks for the updates.

**Pull request merged**